cmd/kured/main.go

```diff
@@ -101,9 +105,20 @@ func newCommand(name string, arg ...string) *exec.Cmd {
 	return cmd
 }
 
+func sentinelCommand() *exec.Cmd {
```

Comment:
do we really need this? Can't we just pass a sentinel command that's defaulting to the current systemctl command instead? We know the first part will always be nsenter -m /proc/1/ns/mnt , so the user could simply pass "systemctl reboot" as an argument?